### DISCUSSION TITLE
Access NuxtApp in defineEventHandler

### DISCUSSION THREAD

**Discussion by xtoolkit** [nuxt/nuxt#16193] [Questions]
Hello everyone

Can i use Nuxt instance with `useNuxtApp()` in `defineEventHandler`? or we can create share a service between server and client?

**Comment by IsraelOrtuno** [comment]
Not sure about that but curious about your use case. What do you need from `useNuxtApp()` in the server?

**Comment by TheAlexLichter** [comment]
You cannot use the nuxt app in nitro as Nuxt will be initialized *after* Nitro handles the request. You *can* pass over data though.
